lib/rihanna/migration/upgrade.ex

```diff
@@ -72,6 +72,17 @@ defmodule Rihanna.Migration.Upgrade do
       """,
       """
       ALTER TABLE #{table_name} DROP COLUMN rihanna_internal_meta;
+      """,
```

Comment:
I'm reading this as that we're missing the statement to drop `$TABLE_enqueued_at_id`. Could you add this as well please? :)

Reply:
To drop the index?